docs/cis_self_assessment.md

```diff
@@ -2341,38 +2107,15 @@ Note: This recommendation only applies if you let kubelets get their certificate
 
 **Result:** Pass
```

Comment:
In order to stay honest, lets change this one to not applicable, since we do our own cert mgmt.